packages/zoe/test/zoeTestHelpers.js

```diff
@@ -41,7 +41,7 @@ export const assertPayoutAmount = async (
   label = '',
 ) => {
   const amount = await issuer.getAmountOf(payout);
-  assertAmountsEqual(t, amount, expectedAmount, label);
+  await assertAmountsEqual(t, amount, expectedAmount, label);
```

Comment:
Just checking my understanding here: Would `return` work as well? `assertAmountsEqual()` isn't async.  Oh, I guess your comments say that this isn't normally required and shouldn't be. 

Why does an await here improve our ability to chain back through the promises?

Reply:
I'm not actually sure. In fact, I find it quite strange. I added these awaits from reasoning by analogy with examples I saw on some web pages when searching about v8's deep stack support. (TODO I should find these again and add links.) This would seem to indicate that without these awaits, v8 doesn't have the bookkeeping needed to trace back these deep stacks. 

***BUT*** the way I found these places to put the await was by using the vscode debugger that showed these deep stacks in the absence of the awaits. The vscode debugger gets this info only from v8, which means that v8 actually knows the deep stacks without these awaits, but will only show them if you use these awaits.

@syg, do you know what's going on here?

Reply:
But I want to emphasize that I do *not* recommend adding additional awaits for the sake of debugging unless being very confident that the awaits are not causing harm. For an await that is not otherwise necessary, adding it is indeed likely to be harmful, so this will be rare.